pkg/client/client_test.go

```diff
@@ -188,3 +188,19 @@ func TestHttpStatusCodes(t *testing.T) {
 		})
 	}
 }
+
+func TestAddEventsEndpointUrlTrailingSlashIsHandledCorrectly(t *testing.T) {
```

Comment:
why dont we split this into two tests?

Reply:
Was think about it, I can do it.

Reply:
Changed in 064dbca057c7e7517d54f9b7a7f6e93742ea793e.